src/css/_typography.scss

```diff
@@ -94,6 +94,26 @@ ol[class] {
     margin-top: 0;
   }
 
+  & > h1 {
+    font-size: 32px;
+    line-height: 48px;
+  }
+
+  & > h2 {
+    margin-top: 32px;
+    font-size: 24px;
+    line-height: 40px;
+
+    &:first-of-type {
```

Comment:
I guess this is for "the title" of the page, but It seems odd using it for `h2`. Can we discuss that?